src/Microsoft.DotNet.ProjectModel.Server/ApplicationContext.cs

```diff
@@ -0,0 +1,455 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using Microsoft.DotNet.ProjectModel.Resolution;
+using Microsoft.DotNet.ProjectModel.Server.Helpers;
+using Microsoft.DotNet.ProjectModel.Server.InternalModels;
+using Microsoft.DotNet.ProjectModel.Server.Models;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.ProjectModel.Server
+{
+    public class ApplicationContext
```

Comment:
Rename this, it should be a variant of `ProjectContext` like `RemoteProjectContext`